language/ir-testsuite/tests/effects/publish_then_unpublish.mvir

```diff
@@ -0,0 +1,27 @@
+module M {
```

Comment:
Can these be written as source lang tests? I think we should avoid adding new IR tests unless we can't avoid it...

Reply:
Other tests of the same class are written in IR so I'd decided to keep it that way in this PR. How about I put up another PR to migrate them all at once?